packages/amplify_datastore_plugin_interface/test/testData/PostAuthComplex.dart

```diff
@@ -21,8 +21,16 @@ import 'package:flutter/foundation.dart';
 class PostAuthComplex extends Model {
   static const classType = const PostAuthComplexType();
   final String id;
-  final String title;
-  final String? owner;
+  final String? _title;
+  final String? _owner;
+
+  String get title {
```

Comment:
Again, I guess I'm a little confused as `_title` is optional but not sure why this does not have `String?` syntax like owner below.

Reply:
see answer above.